packages/core/src/modules/credentials/formats/__tests__/JsonLdCredentialFormatService.test.ts

```diff
@@ -149,12 +147,12 @@ const inputDoc = {
     },
     alumniOf: 'oops',
   },
-}
+})
 const verificationMethod = `8HH5gYEeNc3z7PYXmd54d4x6qAfCNrqQqEB3nS7Zfu7K#8HH5gYEeNc3z7PYXmd54d4x6qAfCNrqQqEB3nS7Zfu7K`
-const credential = JsonTransformer.fromJSON(inputDoc, W3cCredential)
+// const credential = JsonTransformer.fromJSON(inputDoc, W3cCredential)
 
-const signCredentialOptions: SignCredentialOptionsRFC0593 = {
-  credential,
+const signCredentialOptions: SignCredentialOptionsRFC0593AsJson = {
+  credentialAsJson: inputDocAsJson,
```

Comment:
can we make the input just `credential`? 

```suggestion
  credential: inputDocAsJson,
```

Reply:
Yes, this was really to aid in development, as it is nice to be able to see the new field distinguished from the old one in certain places.